microbenchmarks/results.txt

```diff
@@ -0,0 +1,89 @@
+# on macbook pro
```

Comment:
@dlg99 One thing to note is that the SSE4.2 instructions are using a JNI module and that's only compiled for linux in the released artifacts for `pulsar-checksum`.